exercises/isogram/isogram-example.apl

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env apl --script
+
+version ← '1.2.0' ⍝ isogram exercise, GNU-APL track, exercism
+
+alphabet ← 'abcdefghijklmnopqrstuvwxyz'
+ALPHABET ← 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+
+counts ← {+/ ⍺ ∘.= ⍵}
+
+isIsogram ← {2 > ⌈/ (alphabet counts ⍵) + (ALPHABET counts ⍵)}
```

Comment:
I'm not at all sure that I like this implementation; I think I'd prefer to normalize case and count. But I'll think about it further.